onnxruntime/core/providers/vitisai/imp/tensor_proto.cc

```diff
@@ -16,6 +16,12 @@ gsl::span<const char> tensor_proto_as_raw(
     std::vector<uint8_t> unpacked_tensor;
     auto s = onnxruntime::utils::UnpackInitializerData(tensor, onnxruntime::Path(), unpacked_tensor);
```

Comment:
What happens if the initializer is from External data location? 
i.e. https://github.com/microsoft/onnxruntime/blob/8d48d3e9cce4a7b5640051899f824c69a6a67c62/onnxruntime/core/framework/tensorprotoutils.cc#L1509

Reply:
It is known issue documented in our internal issue system. 
For now, there is a limitation that data must resides at the same location.
The reason it is not fixed yet is because there would be a lots of diffs.

But we will fix it eventually.